src/components/content-tab-overview.tsx

```diff
@@ -234,15 +237,43 @@ export function ContentTabOverview( { selectedSite }: ContentTabOverviewProps )
 					className={ cx(
 						'w-full min-h-40 max-h-60 overflow-hidden rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex items-center justify-center',
 						loading && skeletonBg,
-						isThumbnailError && 'border-none'
+						isThumbnailError && 'border-none',
+						siteRunning && 'hover:border-a8c-blueberry duration-300'
 					) }
 				>
 					{ isThumbnailError && ! loading && (
 						<div className="flex items-center justify-center w-full h-full leading-5 text-a8c-gray-50">
 							{ __( 'Preview unavailable' ) }
 						</div>
 					) }
-					{ ! loading && (
+					{ ! loading && siteRunning && (
+						<button
+							aria-label={ __( 'Open WP Admin' ) }
+							className={ cx( `relative group` ) }
```

Comment:
I couldn't make the focus state work and it was yellow all the time. How to do it correctly?

Reply:
Did you try using something like Button component that we have: https://github.com/Automattic/studio/blob/trunk/src/components/button.tsx ? It would take care of all these styles including focus so you would not need to manually fix them

Reply:
Yes, and it was even worse. I've just pushed the fix for the outline issue.

Reply:
Hahaha, it does not surprise me 😅 

Ideally I would love for us to use the components we have instead of adding the button manually but I have not explored this solution in detail.